src/locales/fr/messages.json

```diff
@@ -60,7 +60,7 @@
     "message": "Élément partagé"
   },
   "shareDesc": {
-    "message": "Choisissez une organisation avec laquelle vous voulez partager cet élément. Le partage transfère la propriété de l'élément à l'organisation. Vous ne serez plus le propriétaire direct de cet élément après le partage."
+    "message": "Choisissez un dossier avec lequel vous voulez partager cet élément. Le partage transfère la propriété de l'élément au dossier. Vous ne serez plus le propriétaire direct de cet élément après le partage."
```

Comment:
This wording is here because we still display the Bitwarden native interface, but in the future it will be "hidden" as folders won't be considered as "shared" until we use the Cozy sharing dialog.

So if you are ok I propose to let this line untouched now and it will be handled in future tasks.